third_party/pcsc-lite/naclport/js_client/src/context.js

```diff
@@ -115,23 +116,33 @@ goog.exportProperty(Context.prototype, 'logger', Context.prototype.logger);
 /**
  * Starts the initialization process by initiating a connection to the server
  * App.
+ *
+ * If the optional message channel is passed, then it is used instead of opening
```

Comment:
Turns out, it's not allowed to open a port from a background page to itself. So I had to write a fake message channel and extend this class to be able to receive it instead of opening the port.